olive/passes/pytorch/qat_utils.py

```diff
@@ -168,7 +169,7 @@ def create_dummy_input(self):
         # dummy inputs
         dummy_inputs = []
         for input_shape, input_type in zip(self.config.input_shapes, input_types):
-            dummy_inputs.append(torch.zeros(input_shape, dtype=input_type))
+            dummy_inputs.append(torch.zeros(input_shape, dtype=input_type, device=Device.CPU))
```

Comment:
better to postpone the data to device when it is used.

Reply:
Done.